### ISSUE TITLE
Open A11y Pilot: What is the target range per document

### ISSUE THREAD

**Issue by nschonni** [mgifford/a11y-contracting#8]
> programmatically suggest alternative text for images (.gif, .png, .svg, and .jpg).
programmatically generate transcripts of voice audio recordings;
programmatically generate described video, transcripts, closed captioning for video content created in both official languages;
programmatically generate animated American Sign Language or Quebec Sign Language from spoken word audio or video;

Picking on these because I believe there is machine learning platforms that may be able to produce this content. The quality may be variable, and some may require intense compute resources. EX: delivering the solution under budget may be achievable, but it the solution costs $100 in compute cost per document, it isn't feasible.

**Comment by mgifford** [comment]
I was walking these back from the original document - https://github.com/mgifford/a11y-contracting/blob/master/Accessibility-Procurement-Pilot.md

I was concerned to see these issues, which I organized into these two categories:

**Not sure that it is useful:**
- Lack of ability within the system to programmatically generate alternate versions of Digital Assets
available through the Open by Default Pilot Portal in a variety of braille formats such as .BRF and
.BRL in both official languages;
- Lack of ability within the system to generate animated American Sign Language or Quebec Sign
Language from spoken word audio or video;

**.Not possible (to a meaningful level) :** 
- Lack of ability within the system to automatically transform Digital Assets created in both official
languages, into Digital Assets which conform to a minimum of WCAG 2.0 AA. Digital assets which
additionally conform to WCAG 2.1 and/or EN 301 549 would be preferred.
- Lack of ability within the system to programmatically add system generated alt text / descriptive text in both official languages to images;
- Lack of ability within the system to programmatically generate transcripts of English and French
voice audio recordings;
- Lack of ability within the system to programmatically generate described video, transcripts, closed
captioning for video content created in both official languages;
- The absence of the ability for the system to programmatically reduce the reading grade level of the
content to a target of grade 8, in accordance with WCAG 2.0 AAA; and,

Fascinating though that the government could potentially pay $300k to build an app that successfully suggests alt text for images at a cost of $100/image.  Would that be considered a success?  How accurate does that image definition need to be at what price?

**Comment by mgifford** [comment]
Denis has a good post on AI https://www.24a11y.com/2017/five-ways-in-which-ai-changes-the-face-of-web-accessibility/
